conda.rb

```diff
@@ -72,4 +84,13 @@ def download_channeldata(channel, domain)
     url = "https://#{domain}/#{channel}/channeldata.json"
     HTTParty.get(url).parsed_response
   end
+
+  def channel_and_name_from_key(key)
+    # Split apart the channel and name from the redis key `packages:channel/name`
+    channel_name = key.rpartition(":").last.rpartition("/")
```

Comment:
You can condense this down to one line. Something like `channel, _separator, name = key.rpartition(":").last.rpartition("/")`

Reply:
ah right because rpartition only does two splits,  derp